site/en/docs/handbook/how-to/add-an-article/index.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: 'layouts/doc-post.njk'
+title: Add an article
+description: 'Add a new post to /articles'
+date: 2022-05-19
+---
+
+## Before you start
+
+Make sure your content is an article, and not reference documentation or a blog post. See [where to publish](/docs/handbook/where-to-publish) for more information.
+
+## Clone the sample post
+
+Duplicate the example post located in `site/en/articles/_example/` and
+rename the folder to match whatever slug you want to use for your url.
+
+Posts support markdown syntax with some additional features and shortcodes.
+
+## Metadata
+
+Each `index.md` file starts with YAML Front Matter. See
+`site/en/articles/_example/` for documentation of all supported metadata.
+
+Articles and blog posts share the same template and are identical other than location at the moment.
+
+### Hero images
+
+Hero images should be at least 1920 x 960. Use jpg unless you have a specific
+need for png or svg. Our image CDN will handle compressing and converting the
+image to newer formats such as webp so you don't have to do that in advance.
```

Comment:
```suggestion
Hero images should be at least 1920 x 960. Use JPG unless you have a specific
need for PNG or SVG. Our image CDN will handle compressing and converting the
image to newer formats such as WebP so you don't have to do that in advance.
```

Reply:
This is duplicated from the other how-tos, which likely date from the launch of this site. I should go update those in another PR - thanks!